lib/test/karma.js

```diff
@@ -155,7 +166,9 @@ function karmaRun(reporters, bannerWidth) {
   var options = {
     configFile: undefined
   };
-  reporters = reporters || [];
+  if (reporters.constructor !== Array || reporters.length === 0) {
+    throw new Error('No reporters specified');
```

Comment:
In general, do we want to kill the process with an error when no reporter is specified?